### ISSUE TITLE
Remove post from all timelines

### ISSUE THREAD

**Issue by okpierre** [jointakahe/takahe#394]
Ability for admin/mod to remove certain post from **all** timelines

-admin/mod sees inappropriate post in timeline
-click post to review
-remove single post from **all** timelines (home/local/federated)
-all posts can still be seen in user profile
-user may have only one "bad" post while others are "safe"

Looked at https://docs.jointakahe.org/en/latest/features/#features but didn't see something similar listed

**Comment by andrewgodwin** [comment]
Yeah, good call - we need the ability to somehow "hide" it entirely without deleting (so that it won't reappear when another copy comes in via federation).
